### PR TITLE
Removes Django, Docker (closes #1608).

### LINKED ISSUE #1608: Exorcise Django and Docker
gif-4

### PR DESCRIPTION
**Pull request by chuckharmston**
WIP so you can follow along!

And by "you" I mean "CircleCI".

### CODE REVIEW

**Comment by chuckharmston** [comment]
Updating `gulp-sourcemaps` to 1.7.x per [this advice](https://github.com/floridoo/gulp-sourcemaps/issues/238)

**Comment by chuckharmston** [comment]
No longer WIP. Build down to 3:11. Ready for review!

File: circle.yml [hunk 1]
```diff
@@ -39,12 +36,11 @@ dependencies:
...
     - ./bin/circleci/build-addon.sh
     - ./bin/circleci/build-frontend.sh
-    - '[[ " $TESTPILOT_STATIC_BRANCHES " =~ " $CIRCLE_BRANCH " ]] || ./bin/circleci/build-server.sh'
 
 test:
   override:
-    - '[[ " $TESTPILOT_STATIC_BRANCHES " =~ " $CIRCLE_BRANCH " ]] || ./bin/circleci/run-server-unit-tests.sh'
-    # - ./bin/circleci/run-integration-tests.sh
+    - echo "Skipping integration tests."
+    # - ./bin/circleci/run-integration-tests.shi
```
Comment (pdehaan) [inline circle.yml]: *.shi?

File: frontend/tasks/scripts.js [hunk 2]
```diff
@@ -8,7 +8,6 @@ const gulpif = require('gulp-if');
 const gutil = require('gulp-util');
 const eslint = require('gulp-eslint');
 const source = require('vinyl-source-stream');
-const sourcemaps = require('gulp-sourcemaps');
```
Comment (dannycoates) [inline frontend/tasks/scripts.js:11]: I'd like to see this come back now that upstream is fixed

**Comment by lmorchard** [comment]
Going to take a look through this today. Should make #1522 way easier too

File: circle.yml [hunk 3]
```diff
@@ -39,12 +36,11 @@ dependencies:
     - ./bin/circleci/build-version-json.sh
     - ./bin/circleci/build-addon.sh
     - ./bin/circleci/build-frontend.sh
-    - '[[ " $TESTPILOT_STATIC_BRANCHES " =~ " $CIRCLE_BRANCH " ]] || ./bin/circleci/build-server.sh'
 
 test:
   override:
-    - '[[ " $TESTPILOT_STATIC_BRANCHES " =~ " $CIRCLE_BRANCH " ]] || ./bin/circleci/run-server-unit-tests.sh'
-    # - ./bin/circleci/run-integration-tests.sh
+    - echo "Skipping integration tests."
```
Comment (lmorchard) [inline circle.yml:42]: :+1: Good to leave that there as a reminder for sure

File: circle.yml [hunk 4]
```diff
@@ -39,12 +36,11 @@ dependencies:
...
+    # - ./bin/circleci/run-integration-tests.shi
 
 # appropriately tag and push the container to dockerhub
 deployment:
```
Comment (lmorchard) [inline circle.yml:46]: Docker-dependent stuff below here that will break on merge to master. Should delete the `hub_*` deployment cases, leaving the `static_deployment` bit. 

I'd also change the branch from `development-static` to `master`, which will deploy to testpilot-static.dev.mozaws.net. But, I can follow up on that in #1522

File: frontend/tasks/scripts.js (hunk 2 above)
Comment (lmorchard) [reply frontend/tasks/scripts.js:11]: Does this still need to be removed, or did #1613 & #1614 resolve the issue?

**Review by lmorchard — CHANGES_REQUESTED** [review]
:+1: but for 2 things - removing the Dockerhub deployment tasks from `circle.yml` and re-adding the sourcemaps stuff that I assume was fixed with #1614

**Comment by chuckharmston** [comment]
@lmorchard Changes made, build passed.

**Review by lmorchard — APPROVED** [review]

**Comment by lmorchard** [comment]
💥

**Pull request merged**